src/views/List.jsx

```diff
@@ -12,6 +16,7 @@ export function List({ data }) {
 				 * using the `ListItem` component that's imported at the top
 				 * of this file.
 				 */}
+				{listItems}
```

Comment:
@aberger3647 @arbitrary-berry small nit: intead of storing as a variable you could do this also:

```
return (
    <>
        <p>
           {data.map((data) => (
              <ListItem key={data.id} name={data.name} />
           ))}
        </p>
      <>
```

Either approach is fine and no need to change this, just wanted to point out how it's usually written.

Reply:
Makes sense! No need to create a variable here. Thanks.